static/js/locationsTableUpdate.js

```diff
@@ -78,4 +66,7 @@ function cellUpdate(col, data) {
     }
 }
 
-//function tableLabelUpdate(rack):
+function goBack() {
+    window.history.back();
```

Comment:
you shouldn't need a js function to do this, if you're passing the location id to the template then use the _{{ url_for(route, location_id=location.id) }}_